### PR TITLE
Resolved a few PR comments

### PR DESCRIPTION
**Pull request by satwikgoyal**
Added a home.js for the home component, so that we can now route the pages from App.js. Adding routing through App.js. Added box dimensions for medium and small screen sizes, however, a different UI design should be implemented later. Made a useWindowSize() in gatheredList.js that can later be used to make the entire UI screen size responsive.  Made the menuItems dynamic for later use in notesMenu.js using the .map function.

### CODE REVIEW

File: frontend/src/pages/components/gatheredList.js [hunk 1]
```diff
@@ -51,35 +51,57 @@ export default function InfoGatheredList({pages}) {
...
+  const isSmall = width < 640;
+  const isMedium = width < 1024; //could also be 1007 instead of 1024 depending on standard used
+  const margin_left = isSmall? 1 : (isMedium ? 2 : 8);
+  const title_fontSize = isSmall? '8px' : (isMedium ? '12px' : '16px'); //use it later when making it suitable for medium and small sizes
+
+
   return (
     <div className={classes.root}>
-      <Box mt = {6} ml = {8}>
+      <Box mt = {6} ml = {margin_left}>
```
Comment (rasdy) [inline frontend/src/pages/components/gatheredList.js]: I think you could achieve a similar thing by just using a percentage value, like `ml = "20%"`. If that doesn't have the effect you're looking for, then disregard.

File: frontend/src/pages/components/notesMenu.js [hunk 2]
```diff
@@ -47,6 +47,12 @@ export default function CustomizedMenus() {
     setAnchorEl(null);
   };
 
+  const gatheredInformation = ["Assignment", "Stakeholder 1", "Stakeholder 2"]
```
Comment (rasdy) [inline frontend/src/pages/components/notesMenu.js:50]: What is the CustomizedMenus component used for? It doesn't have to do with the gatheredInfo component does it?

**Review by rasdy — CHANGES_REQUESTED** [review]

File: frontend/src/pages/components/notesMenu.js (hunk 2 above)
Comment (satwikgoyal) [reply frontend/src/pages/components/notesMenu.js:50]: I am not sure where we will exactly use it. I think Song and Tim made this file. However, I still made the change to resolve the comment.

File: frontend/src/pages/components/gatheredList.js (hunk 1 above)
Comment (satwikgoyal) [reply frontend/src/pages/components/gatheredList.js]: Okay, I will do that.

Comment (satwikgoyal) [reply frontend/src/pages/components/gatheredList.js]: Yes, it works, made the change.

**Pull request merged**